projects/DomeViewer/Assets/Modules/FileSystem/OBBHandler.cs

```diff
@@ -0,0 +1,83 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System.IO;
+
```

Comment:
File is missing namespace. See the other files for reference

Reply:
namespace is added and unneeded references removed